services/common/src/components/common/ActionMenu.spec.tsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import ActionMenu, { ActionMenuButton, deleteConfirmWrapper, generateActionMenuItems } from './ActionMenu';
+import { Modal } from 'antd';
+
+jest.mock('antd', () => {
+    const originalModule = jest.requireActual('antd');
+    return {
+        ...originalModule,
+        Modal: {
+            ...originalModule.Modal,
+            confirm: jest.fn(),
+        },
+    };
+});
+
+describe('ActionMenu', () => {
```

Comment:
oh nice test addition! Yeah... this probably should have already been in there... I'll have to double-check the output but I bet this takes out a lot of the uncovered lines in coverage reports.